src/visumorph/change_hue.py

```diff
@@ -0,0 +1,32 @@
+def change_hue(image, delta_hue):
+    """
+    Adjusts the hue of the given image.
+
+    This function modifies the hue of each pixel in the input image by the specified amount (delta_hue),
+    changing the overall color composition without altering luminance or saturation.
+
+    Parameters:
+    image (Image Object): The image to be processed. Must be a valid Image Object as defined in the VisuMorph package.
+                          The image should be in a color format recognizable by VisuMorph (e.g., RGB).
+    delta_hue (float): The amount by which to shift the hue component in the HSL color space. The value can be either
+                       positive or negative, representing a clockwise or counterclockwise rotation of the hue, respectively.
+
+    Returns:
+    Image Object: A new Image Object with the hue adjusted as per the delta_hue parameter. The returned image has the same
```

Comment:
Same issue with the Parameters - Please follow the [format](https://numpydoc.readthedocs.io/en/latest/format.html#returns)

Reply:
Resolving this as commit abea67a3ba0a9fc4f1b9d9dd5f1a0670786f717d addressed this issue